mslib/mscolab/conf.py

```diff
@@ -76,6 +76,12 @@ class default_mscolab_settings:
     # used to generate the password token
     SECURITY_PASSWORD_SALT = secrets.token_urlsafe(16)
 
+    # Max allowed file size for uploading user profile image
+    MAX_IMAGE_SIZE = 1 * 1024 * 1024  # 1MB
+
+    # Allowed file extensions for uploading user profile image
+    ALLOWED_EXTENSIONS = {'png', 'jpg', 'jpeg'}
```

Comment:
that is confusing for the chat attachments, because we can share there also ftml files, csv files etc.
May be name this ALLOWED_IMAGE_EXTENSION, so it can be reused on other places.

Reply:
I am also not sure if that is needed, when the server stores the image as file, it may be only one type, e.g. png for the users picture